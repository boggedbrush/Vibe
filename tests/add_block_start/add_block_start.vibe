# VibeSpec: 1.2
patch_type: add_block
file: hello.py
position: start
--- code: |
  # >>> Inserted at top
  import logging
  logging.basicConfig(level=logging.INFO)
