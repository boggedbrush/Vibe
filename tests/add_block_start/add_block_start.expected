# >>> Inserted at top
import logging
logging.basicConfig(level=logging.INFO)

# hello.py


def greet(name):
    print(f"Hello, {name}!")


class Greeter:
    def __init__(self, name):
        print(f'Hello {name}')
